beacon_node/beacon_chain/src/attester_per_slot_service.rs

```diff
@@ -0,0 +1,122 @@
+use crate::{BeaconChain, BeaconChainTypes};
+use slog::{debug, error};
+use slot_clock::SlotClock;
+use std::sync::Arc;
+use task_executor::TaskExecutor;
+use tokio::time::sleep;
+use types::{BeaconState, RelativeEpoch, Slot};
+
+/// Spawns a routine which produces an unaggregated attestation at every slot.
+///
+/// This routine will run once per slot
+pub fn start_attester_per_slot_service<T: BeaconChainTypes>(
+    executor: TaskExecutor,
+    chain: Arc<BeaconChain<T>>,
+) {
+    // TODO: AI(joel) Only run the service if validator monitor is enabled
+    // Paul has made a refacto of that bit in another PR, I will rebase
+    // once it's merged
+    executor.clone().spawn(
+        async move { attester_per_slot_service(executor, chain).await },
+        "attester_per_slot_service",
```

Comment:
```suggestion
        "attestation_simulator_service",
```

I'd suggest calling this feature the "attestation simulator" and trying to use the word "simulator" in all the logs, just to avoid confusion with actual, validator-created attestations.